libs/@local/hash-isomorphic-utils/src/blocks.ts

```diff
@@ -124,7 +125,10 @@ const transformBlockConfig = ({
     variants,
     icon: deriveAbsoluteUrl({ baseUrl, path: metadata.image }),
     image: deriveAbsoluteUrl({ baseUrl, path: metadata.icon }),
```

Comment:
Is this an unrelated mistake? :D
```suggestion
    icon: deriveAbsoluteUrl({ baseUrl, path: metadata.icon }),
    image: deriveAbsoluteUrl({ baseUrl, path: metadata.image }),
```

Reply:
That does look like a bug to me haha
Fix presumed bug